### PR TITLE
added rumble to all ArmToPoint commands

### PR DESCRIPTION
**Pull request by Naher123**
added rumble to all ArmToPoint commands

### CODE REVIEW

File: smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java [hunk 1]
```diff
@@ -382,7 +382,7 @@ private void bindCommandsToCoDriverXboxButtons()
         this.coDriverController.rightTrigger().onTrue(
           new ParallelCommandGroup(
             new ArmToPointCommand(
-              this.subsystemCollection.getArmSubsystem(),
+              this.subsystemCollection.getArmSubsystem().andThen(new RumbleCommand()),
```
Comment (mmcadamsms) [inline smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java:385]: This won't build.  Please move the .andThen down to follow the ParallelCommandGroup

File: smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java [hunk 2]
```diff
@@ -394,7 +394,7 @@ private void bindCommandsToCoDriverXboxButtons()
         this.coDriverController.leftTrigger().onTrue(
           new ParallelCommandGroup(
             new ArmToPointCommand(
-              this.subsystemCollection.getArmSubsystem(),
+              this.subsystemCollection.getArmSubsystem().andThen(new RumbleCommand()),
```
Comment (mmcadamsms) [inline smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java:397]: Same as above ...

File: smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java [hunk 3]
```diff
@@ -382,7 +382,7 @@ private void bindCommandsToCoDriverXboxButtons()
         this.coDriverController.rightTrigger().onTrue(
```
Comment (mmcadamsms) [inline smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java:382]: you need an import at the top of this file

**Review by mmcadamsms — CHANGES_REQUESTED** [review]

File: smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java (hunk 2 above)
Comment (Naher123) [reply smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java]: done

File: smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java (hunk 3 above)
Comment (Naher123) [reply smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java:384]: done

**Review by mmcadamsms — APPROVED** [review]
lookin' good ... ship it!

**Pull request merged**